src/display/font_loader.js

```diff
@@ -71,13 +79,61 @@ class FontLoader {
     }
   }
 
+  async loadSystemFont(info) {
+    assert(
+      !this.disableFontFace,
+      "loadSystemFont shouldn't be called when `disableFontFace` is set."
+    );
+
+    const { loadedName, src, style } = info;
+    if (this.#systemFonts.has(src)) {
+      return;
+    }
+    this.#systemFonts.add(src);
+    if (this.isFontLoadingAPISupported) {
+      const fontFace = new FontFace(loadedName, src, style);
+      this.addNativeFontFace(fontFace);
+      try {
+        await fontFace.load();
+      } catch {
+        warn(
+          `Cannot load system font: ${loadedName} for style ${style.style} and weight ${style.weight}.`
+        );
+        this.removeNativeFontFace(fontFace);
+      }
+      return;
+    }
+
+    if (this._document) {
+      const rule =
+        "@font-face { " +
+        `font-family: ${loadedName};` +
+        `src: ${src};` +
+        `font-style: ${style.style};` +
+        `font-weight: ${style.weight}; }`;
+      this.insertRule(rule);
+    }
+  }
+
+  get readyPromise() {
+    if (!this.isFontLoadingAPISupported || !this._document) {
+      return Promise.resolve();
+    }
+    return this._document.fonts.ready;
+  }
+
```

Comment:
We shouldn't need this anymore, nor any of the changes in the `src/display/api.js` file, right?